### PR TITLE
AoU mitochondria variant calling workflow

### PR DESCRIPTION
**Pull request by eviewan**
In this PR, I added a workflow "MitochondriaProcessing" to call variants from mitochondrial reads. In this workflow, I subset chrM reads from bam file and used Hifiasm and Minimap2 for genome assembly and alignment. Genome assembly evaluation reports were computed with Quast.

### CODE REVIEW

File: wdl/AoU_Mitochondria.wdl [hunk 1]
```diff
@@ -0,0 +1,126 @@
+version 1.0
+
+
+##########################################################################################
```
Comment (SHuang-Broad) [inline wdl/AoU_Mitochondria.wdl:4]: I guess this is carried over from the other workflows, but I find these not good practice anymore given that we have `meta` section for workflows and tasks.

**Review by SHuang-Broad — CHANGES_REQUESTED** [review]

File: wdl/AoU_Mitochondria.wdl [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    String PL = Parsing.PL
+    String PU = Parsing.PU
+    String RG = "@RG\\tID:~{ID}\\tSM:~{SM}\\tPL:~{PL}\\tPU:~{PU}"
+
+    call HA.Hifiasm as Hifiasm {input: reads = BamToFastq.reads_fq, prefix = prefix}
+    call AR.Minimap2 as Minimap2 {input: reads = [Hifiasm.fa], ref_fasta = ref_fasta, map_preset = "map-hifi", RG = RG}
+    call Quast.Quast as Quast {input: ref = ref_fasta, assemblies = [Hifiasm.fa]}
+
+
+    call CallAssemblyVariants.CallAssemblyVariants as  CallAssemblyVariants {input: asm_fasta = Hifiasm.fa,
```
[the remark this answers is not in the document]
Comment (ewanbroad) [reply wdl/AoU_Mitochondria.wdl:90]: move 79-82 into 83

File: wdl/AoU_Mitochondria.wdl [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+            File report_html = Quast.report_html
+            File report_txt = Quast.report_txt
+            File report_pdf = Quast.report_pdf
+            Array[File] quast_plots = Quast.plots
+
+            File paf = CallAssemblyVariants.paf
+            File paftools_vcf = CallAssemblyVariants.paftools_vcf}
+}
+
```
[the remark this answers is not in the document]
Comment (ewanbroad) [reply wdl/AoU_Mitochondria.wdl:111]: fixed

File: wdl/AoU_Mitochondria.wdl [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+## A workflow
+##########################################################################################
+import "tasks/Utils.wdl" as Utils
+import "tasks/Hifiasm.wdl" as HA
+import "tasks/AlignReads.wdl" as AR
+import "tasks/Quast.wdl" as Quast
+import "tasks/CallAssemblyVariants.wdl" as  CallAssemblyVariants
+
+task RG_Parsing {
+    input{
```
[the remark this answers is not in the document]
Comment (ewanbroad) [reply wdl/AoU_Mitochondria.wdl:14]: fixed

File: wdl/AoU_Mitochondria.wdl [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+        bootDiskSizeGb:         10
+        preemptible:            3
+        maxRetries:             2
+        docker:                 "us.gcr.io/broad-dsp-lrma/lr-basic:0.1.1"
+    }
+}
+
+
+workflow MitochondriaProcessing{
+    input{
```
[the remark this answers is not in the document]
Comment (ewanbroad) [reply wdl/AoU_Mitochondria.wdl:50]: fixed

File: wdl/AoU_Mitochondria.wdl (hunk 1 above)
Comment (ewanbroad) [reply wdl/AoU_Mitochondria.wdl:4]: fixed

**Pull request closed without merging**